azure-pipelines.yml

```diff
@@ -38,7 +38,7 @@ stages:
         vmImage: windows-latest
       steps:
       - checkout: self
-      - powershell: eng/enforce-issue.ps1 -PullRequestNumber $(System.PullRequest.PullRequestNumber)
+      - powershell: eng/enforce-issue.ps1 -PullRequestNumber $(System.PullRequest.PullRequestNumber) -RepositoryName dotnet/arcade-services
```

Comment:
Could we get the repo name from the AzDO variables rather than hardcoding it? It doesn't look right.